app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,20 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.JavaHelpFrontend"
         tools:targetApi="31">
+        <activity
+            android:name=".frontend.PRegActivity"
+            android:exported="false">
+            <meta-data
+                android:name="android.app.lib_name"
+                android:value="" />
+        </activity>
+        <activity
+            android:name=".frontend.FrontPageActivity"
+            android:exported="false">
+            <meta-data
+                android:name="android.app.lib_name"
+                android:value="" />
+        </activity>
         <activity
             android:name=".frontend.MainActivity"
             android:exported="true">
```

Comment:
What is happening here?

Reply:
I think for each activity, you have to declare them in the AndroidManifest, and the PRehActivity and FronPageActivity are activities that we want to link to the Mainactivity